_data/simple-icons.json

```
@@ -6114,6 +6114,19 @@
                 "type": "GPL-2.0-or-later"
             }
         },
+        {
+            "title": "GNU Free Documentation License",
+            "aliases": {
+                "aka": [
+                    "GFDL"
+                ]
+            },
+            "hex": "2E439B",
+            "source": "https://www.gnu.org/graphics/license-logos.html",
+            "license": {
+                "type": "GFDL-1.3-only"
+            }
```

Would you mind to put `aliases` after `license`?

```suggestion
            "hex": "2E439B",
            "source": "https://www.gnu.org/graphics/license-logos.html",
            "license": {
                "type": "GFDL-1.3-only"
            },
            "aliases": {
                "aka": [
                    "GFDL"
                ]
            }
```